### PR TITLE
Incorporate pprof HTTP debugging handler(#104)

### PR DESCRIPTION
**Pull request by jordy1024**
support pprof debug tool.

### CODE REVIEW

File: web/server.go [hunk 1]
```diff
@@ -108,8 +108,7 @@ func Server() {
 	r.Use(middleware.RealIP)
 	r.Use(middleware.Logger)
 	r.Use(middleware.Recoverer)
-	r.Use(middleware.RedirectSlashes)
-	r.Use(middleware.StripSlashes)
+	r.Mount("/debug", middleware.Profiler())
```
Comment (proditis) [inline web/server.go:111]: Hi @jordy1024, just a quick question, why the Mount "/debug" removes RedirectSlashes and StripSlashes (excuse my ignorance i'm fairly new with Go)

Comment (proditis) [reply web/server.go:111]: Never mind, I just saw your issue :+1:

Comment (jordy1024) [reply web/server.go:111]: > Hi @jordy1024, just a quick question, why the Mount "/debug" removes RedirectSlashes and StripSlashes (excuse my ignorance i'm fairly new with Go)

some issues about  `middleware.Profiler()`: https://github.com/go-chi/chi/issues/262

Maybe it's not the best solution. @proditis

Comment (haoel) [reply web/server.go:111]: Removing them is fine. And could you please test the following URls?

htttp://localhost:8181/sla
htttp://localhost:8181/sla/

htttp://localhost:8181/api/v1/sla
htttp://localhost:8181/api/v1/sla/

Comment (proditis) [reply web/server.go:111]: > Maybe it's not the best solution. @proditis

I cant think of anything wrong with it. As long as our SLA endpoints work I think this is an ok change.

Comment (jordy1024) [reply web/server.go:111]: > Removing them is fine. And could you please test the following URLs?
> 
> http://localhost:8181/sla http://localhost:8181/sla/
> 
> http://localhost:8181/api/v1/sla http://localhost:8181/api/v1/sla/
 
SO sorry, my bad.I'll think of a better solution.

Comment (haoel) [reply web/server.go:111]: Actually, it is really tough problem.  ;-)

1) the URL with a trailing slash or not.
2) if we put the easeprobe behind nginx.  `proxy_pass: http://easeprobe:8181` or  `proxy_pass: http://easeprobe:8181/` could introduce a double slash problem.

**Comment by proditis** [comment]
ping! :smile: 

Hi @jordy1024, I was wondering what is the status of this PR? Its been stall for some time now, anything I can do to help?

**Pull request closed without merging**